notifications_utils/recipients.py

```diff
@@ -482,7 +482,7 @@ def normalise_phone_number(number):
     try:
         list(map(int, number))
     except ValueError as e:
-        raise InvalidPhoneError("Must not contain letters or symbols") from e
+        raise InvalidPhoneError("Mobile number cannot include letters or symbols") from e
```

Comment:
This is better than it used to be, but I'd point out that we do allow users to give us mobile numbers with symbols in, for example `(0)7912345678` contains brackets which are symbols but we strip them out before checking the validation.

Reply:
```suggestion
        raise InvalidPhoneError("Mobile numbers can only include the following characters: 0 1 2 3 4 5 6 7 8 9 ( ) + -") from e
```

Reply:
@karlchillmaid 

This now looks like this:
<img width="700" alt="invalid characters phone error" src="https://github.com/alphagov/notifications-utils/assets/32799090/7d564b0d-3f3b-42fc-8f22-d3ed109359ce">